lib/taskana-core/src/test/java/acceptance/workbasket/CreateWorkbasketAccTest.java

```diff
@@ -57,6 +58,9 @@ public void testCreateWorkbasket()
         Workbasket createdWorkbasket = workbasketService.getWorkbasket("NT1234", "DOMAIN_A");
         assertNotNull(createdWorkbasket);
         assertNotNull(createdWorkbasket.getId());
+
+        Assert.assertEquals(workbasket, createdWorkbasket);
```

Comment:
"Assert." is not needed. See other statements. 

What are the blank lines for?

Reply:
This part is now adjusted with the blank lines removed, i'll wait for new information on the method mentioned above before updating this PR.